modules/videoio/src/cap_images.cpp

```diff
@@ -200,7 +200,7 @@ bool CvCapture_Images::setProperty(int id, double value)
     return false;
 }
 
-static
+// static
 std::string icvExtractPattern(const std::string& filename, unsigned *offset)
```

Comment:
I tried to make as little modification in the source code as possible to simplify merge with #23815 . Maybe it would be better to postpone refactoring? I had some ideas, like multiple pattern support (e.g. `frame_%d_%d.png`), which would require to rework this function even more.